core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -92,26 +92,32 @@
   }
 
   /**
-   * {@link io.grpc.Metadata.Key} for the Content-Type request/response header.
+   * The default port for plain-text connections.
```

Comment:
Originally I wanted to make the header keys be closer in the file to the header values, but I don't feel strongly about this.